crmsh/bootstrap.py

```diff
@@ -856,27 +866,18 @@ def pick_default_value(vlist, ilist):
     mcastport_res = []
     default_ports = ["5405", "5407"]
     two_rings = False
-    default_networks = []
 
-    if _context.ipv6:
-        network_list = []
-        all_ = utils.network_v6_all()
-        for item in all_.values():
-            network_list.extend(item)
-        default_networks = map(utils.get_ipv6_network, network_list)
-    else:
-        network_list = utils.network_all()
-        if len(network_list) > 1:
-            default_networks = [_context.ip_network, network_list.remove(_context.ip_network)]
-        else:
-            default_networks = [_context.ip_network]
-    if not default_networks:
+    local_iplist = utils.ip_in_local(_context.ipv6)
+    len_iplist = len(local_iplist)
+    if len_iplist == 0:
         error("No network configured at {}!".format(utils.this_node()))
 
+    default_ip = [_context.ip_address] + [ip for ip in local_iplist if ip != _context.ip_address]
+
     for i in 0, 1:
```

Comment:
Could we change this `0, 1` by `range(2)`? It looks much better for me hehe

Reply:
OK, I changed all places using `0, 1` to `range(2)`